xep-0369.xml

```diff
@@ -997,7 +1010,7 @@
   </event>
 </message>
 ]]></example>
-      <p>The user that has been added to the channel is identified by the item id of the item added to the Participants node, which is the proxy JID of the new channel participant.  The &lt;participant&gt; element MUST include a &lt;jid&gt; element with the JID of the participant, unless MIX-ANON is being followed to hide participant JIDs.    The &lt;nick&gt; element will not be included at this point, unless it is automatically generated by the channel.   In the likely event that a Nick is subsequently added, an update with the &lt;nick&gt; element will be distributed.
+      <p>The user that has been added to the channel is identified by the item id of the item added to the Participants node, which is the Stable Participant ID of the new channel participant.  The &lt;participant&gt; element MUST include a &lt;jid&gt; element with the JID of the participant, unless MIX-ANON is being followed to hide participant JIDs.    The &lt;nick&gt; element will not be included at this point, unless it is automatically generated by the channel.   In the likely event that a Nick is subsequently added, an update with the &lt;nick&gt; element will be distributed.
```

Comment:
I thought the plan was that you specified your nick when joining? Having a situation where participants are nick-less because their join processing didn't complete seems unhelpful.

Reply:
&lt;Steve&gt; The protocol has setting Nicks independent of Join (it has always been this way). I don’t recollect any plan to change this.

Reply:
0.1 had a note to include the nickname in the join, but the text wasn't yet written.